src/Decorations.ts

```diff
@@ -18,16 +19,16 @@ export default class Decorations {
   decorations: NamedDecoration[];
   decorationMap: DecorationMap;
 
-  constructor() {
-    const fontWidthPx = 8.04;
+  constructor(fontSize: FontSize) {
+    console.log(fontSize)
     const iconWidthPx = 4;
-    const iconWidth = `${iconWidthPx}px`;
-    const iconHeight = "1.7em";
+    const iconPadding = 0;
+    const iconWidth = `${iconWidthPx}`;
```

Comment:
Is this idiomatic?  I usually use `toString` but if this is common practice I'm fine with it

Reply:
Well you did it first so I just continued :D

In js you can concatenate numbers and strings so just `iconWidthPx + "px"` is probably easiest.

Reply:
Ah ok so this was a mistake then. You dropped a `px` in that format string

Reply:
Ah my bad. Will fix that tomorrow :)